modules/getting-cluster-version-and-update-details.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="getting-cluster-version-and-update-details_{context}"]
+= Getting cluster version and update details
+
+After an installation, you can review the cluster version. You can also list the current update channel and review the available cluster updates.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Obtain the cluster version:
++
+[source,terminal]
+----
+$ oc get clusterversion
+----
++
+.Example output
+[source,terminal]
+----
+Cluster version is 4.6.4
```

Comment:
Is this intentionally a stripped down version of the full output?

Example,
```
$ oc get clusterversion
NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
version   4.6.4     True        False         3m26s   Cluster version is 4.6.4
```

Reply:
@staebler thank you for your review.

Thanks for highlighting this. The intention was to include the full output. I have now resolved this.